vivisect/base.py

```diff
@@ -458,6 +458,13 @@ def _handleCOMMENT(self, einfo):
         else:
             self.comments[va] = comment
 
+    def _handleENDIAN(self, einfo):
+        self.bigend = einfo
+        for arch in self.imem_archs:
+            if not arch:
+                continue
+            arch.setEndian(self.bigend)
```

Comment:
Hmm. Should we add something to this to limit it to a specific architecture?

Reply:
i don't think so?
well, possibly?

i'm not sure how to best do that approach, although it would be good for workspaces that include code for multiple architectures....

for now, i think this is good.
however, if you have a better approach system-wide that allows us to set endianness per architecture, and have it tracked through the event subsystem nicely and elegantly, please lay it out.  i'm not pushing back here.  just too much on my mind to prolong this PR any longer on it unless you have something in mind that would be a quick win we agree on :)

i'd just like to get on the same page before i change code ;)

Reply:
Eh. It was more a stray thought for future proofing. But yea, all the other pieces aren't there so we can ignore this for now.